site/server/app/com/fortysevendeg/exercises/controllers/ApplicationController.scala

```diff
@@ -30,12 +33,15 @@ class ApplicationController(
     userProgressOps: UserProgressOps[ExercisesApp],
     T:               Transactor[Task]
 ) extends Controller with AuthenticationModule with ProdInterpreters {
+  implicit def application: Application = Play.current
+
+  lazy val topLibraries: List[String] = application.configuration.getStringList("exercises.top_libraries") map (_.asScala.toList) getOrElse Nil
```

Comment:
Uou! So the Play config seems to be different to the typesafe config library :/

https://github.com/typesafehub/config/blob/ba5e0da6941fd67b779d97c3bd82f1a9dfcfb657/config/src/test/scala/com/typesafe/config/impl/ConfigTest.scala#L584

Hence LGTM!